docker/fedimintd/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+# Important: 127.0.0.1 should be replaced by the external ip address of the machine
+services:
+  fedimintd:
+    image: fedimint/fedimintd:master
+    volumes:
+      - fedimintd_data:/data
+    ports:
+      - "0.0.0.0:8173:8173"
+      - "0.0.0.0:8174:8174"
+    environment:
+      - FM_BITCOIN_RPC_KIND=esplora
+      - FM_BITCOIN_RPC_URL=https://mutinynet.com/api/
+      - FM_BITCOIN_NETWORK=signet
+      - FM_BIND_P2P=0.0.0.0:8173
+      - FM_P2P_URL=fedimint://127.0.0.1:8173
+      - FM_BIND_API=0.0.0.0:8174
+      - FM_API_URL=ws://127.0.0.1:8174
```

Comment:
If someone was to use this in production they should really put a https reverse proxy in front of fedimint (making this a `wss://…` URL). Maybe we should add a warning to that effect? I don't think documenting "how to run NGINX" is really worth it rn.